examples/tree-view/containers/Node.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Component } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions'
+
+class Node extends Component {
+  constructor(props) {
+    super(props)
+    this.handleIncrementClick = this.handleIncrementClick.bind(this)
+    this.handleAddChildClick = this.handleAddChildClick.bind(this)
+  }
+
+  handleIncrementClick() {
+    const { increment, id } = this.props
+    increment(id)
+  }
+
+  handleAddChildClick(e) {
```

Comment:
We bind these functions in the constructor. It's more efficient (which is one of the points of this example) than allocating a function on every render.